redisson/src/main/java/org/redisson/connection/balancer/LoadBalancerManager.java

```diff
@@ -148,9 +149,19 @@ protected void onSuccess(Void value) {
                         }
                     };
                     listener.setCounter(2);
-                    BiConsumer<Void, Throwable> initCallBack = (r, ex) -> {
-                        if (ex == null) {
-                            listener.decCounter();
+                    BiConsumer<Void, Throwable> initCallBack = new BiConsumer<Void, Throwable>() {
+                        private AtomicBoolean initConnError = new AtomicBoolean(false);
+                        @Override
+                        public void accept(Void r, Throwable ex) {
+                            if (ex == null) {
+                                listener.decCounter();
+                            } else {
+                                if (!initConnError.compareAndSet(false, true)) {
+                                    return;
+                                }
+                                entry.closeAllConnections();
```

Comment:
If it's already closed at https://github.com/redisson/redisson/pull/3455/files#diff-637ba54fb9190054f7e2ae91ce22f27a77371807e352055a3131017713a84f23R135

why do we need to close it again?

Reply:
u are right, it's my wrong, i will roll back it